src/controller/index.ts

```diff
@@ -190,6 +194,21 @@ export default class SimulariumController {
         return this.isFileChanging;
     }
 
+    public setOnStreamingChangeCallback(
+        onStreamingChange: (streaming: boolean) => void
+    ): void {
+        this.onStreamingChange = onStreamingChange;
+    }
+
+    private handleStreamingChange(streaming: boolean): void {
+        this.streaming = streaming;
+        this.onStreamingChange(streaming);
+    }
```

Comment:
I don't these these functions are necessary, these kinds of callback are typically used for something in response to something the installing app can't know about, ie when the time has finished updating, or when the file has finished loading and what the data is. all the other `setOn_Change` in the app are in the viewport or decedents of the viewport. The for the controller, the installing app can just call `isStreaming`